google/cloud/bigtable/client.py

```diff
@@ -567,7 +604,20 @@ async def sample_keys(
                 will be chained with a RetryExceptionGroup containing all GoogleAPIError
                 exceptions from any retries that failed
         """
-        raise NotImplementedError
+        operation_timeout = operation_timeout or self.default_operation_timeout
```

Comment:
- Do we want to raise an error if any of the shard queries overlap? Or is it ok to get duplicate rows?
- Do we want this method to retry?

Reply:
yep